func_sig_registry/registry/forms.py

```diff
@@ -41,6 +51,53 @@ def validate_text_signature(self, value):
         return normalize_function_signature(value)
 
 
+class AllSignatureCreateForm(serializers.Serializer):
+    text_signature = serializers.CharField(write_only=True)
+
+    def create(self, validated_data):
+        message_parts = []
+        if 'function_signature' in validated_data['text_signature']:
+            function_signature_tuple = Signature.import_from_raw_text_signature(
+                validated_data['text_signature']['function_signature'],
+                )
+            function_signature = function_signature_tuple[0]
+            message_parts.append('Added function signature {0} for function {1}.'.format(
+                function_signature.bytes_signature.get_hex_display(),
+                function_signature.text_signature,
+            ))
+        if 'event_signature' in validated_data['text_signature']:
+            event_signature_tuple = EventSignature.import_from_raw_text_signature(
+                validated_data['text_signature']['event_signature'],
+            )
+            event_signature = event_signature_tuple[0]
+            message_parts.append('Added event signature {0} for event {1}.'.format(
+                event_signature.get_hex_display(),
+                event_signature.text_signature,
+            ))
+        return {
+            'response_message': ' '.join(message_parts),
+            }
+
+    def validate_text_signature(self, value):
+        validated_signatures = dict()
+        try:
+            normalized_function_signature = normalize_function_signature(value)
+            validated_signatures.update({'function_signature' : normalized_function_signature})
+        except:
+            pass
+
+        try:
+            normalized_event_signature = normalize_event_signature(value)
+            validated_signatures.update({'event_signature' : normalized_event_signature})
+        except:
```

Comment:
bare `try/except` statements are generally :-1: .  They overmatch, catching **any** error which is rarely what you actually want since they can hide bugs that should otherwise be addressed.  This should be updated to catch the exact exeptions you expect to be raised when validation fails.